lilypad/server/services/prompts.py

```diff
@@ -29,6 +29,18 @@ def find_prompts_by_name(
         ).all()
         return record_tables
 
+    def get_next_version(self, project_uuid: UUID, name: str) -> int:
```

Comment:
nit: should we put this on the BaseOrganizationSQLModel?

Reply:
no, there are many models without version